packages/hs-react-ui/src/components/Button/Button.tsx

```diff
@@ -30,18 +30,57 @@ export enum ButtonTypes {
 }
 
 export type ButtonProps = {
+    /**
```

Comment:
Are we going to add these types of comments to all of the props for each of the components for the docs to be created?

Reply:
We will need to do this, but this should be a separate task imo. @wmangimelli could you make a ticket on the Foundry board for this task?

Reply:
Yes, will do.